app/assets/stylesheets/views/article.scss

```diff
@@ -48,6 +48,16 @@
     }
   }
 
+  &__actions {
+    &:empty {
+      display: none;
+    }
+    background: var(--accent-warning-a10);
+    border: 1px solid var(--accent-warning-a10);
```

Comment:
(just a suggestion) - Now that we have this background highlight to draw attention to the actions area, how would you feel about styling the links as links and not buttons? 

From an a11y POV styling links as buttons is bad practice, and this feels like a good opportunity to clean that up. There's also not a lot of contrast between the 'ghost' button hovered state and this yellow:

https://user-images.githubusercontent.com/20773163/133971984-2b4916af-320c-495f-ba04-86131dfe1deb.mp4

Reply:
Is this a11y violation that you would put me in jail for, or just give a ticket? Asking because aesthetically I just liked these looking more like buttons instead link (without underline, colors etc)..

So if it's just a ticket, I'll probably take it :D If it's putting me in jail, I could rethink this area. 

FWIW eventually this will also display a "3 dots" button with dropdown with options (once there's more links there).